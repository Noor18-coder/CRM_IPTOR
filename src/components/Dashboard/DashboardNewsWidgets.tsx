import React from 'react';

const DashboardNewsWidgets: React.FC = () => {
  return (
    <div className="row newsandupdates">
      <div className="col-12">
        <p className="cardsec-title">News &amp; Updates</p>
      </div>

      <div className="col-md-6">
        <div className="card mb-4">
          <div className="row">
            <div className="col-sm-4">
              <div className="img-holder news-img1">
                <p className="newsimg-desc">Seminar 2021 | Live</p>
              </div>
            </div>
            <div className="col-sm-8">
              <div className="news-desc-content">
                <p className="title">
                  Seminar 2021 | Live <span className="news-dt float-right">April 2021</span>
                </p>
                <p className="news-cont">
                  Based on current developments around the COVID-19 spread, growing travel restrictions within the US Iptor customer
                  {/* eslint-disable-next-line react/no-unescaped-entities */}
                  landscape, as well as recommendations from the WHO and global CDC's...
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>

      <div className="col-md-6">
        <div className="card mb-4">
          <div className="row">
            <div className="col-sm-4 col-12">
              <div className="img-holder news-img2">
                <p className="newsimg-desc">Seminar 2021 | Live</p>
              </div>
            </div>
            <div className="col-sm-8">
              <div className="news-desc-content">
                <p className="title">
                  Seminar 2021 | Live <span className="news-dt float-right">April 2021</span>
                </p>
                <p className="news-cont">
                  Based on current developments around the COVID-19 spread, growing travel restrictions within the US Iptor customer
                  {/* eslint-disable-next-line react/no-unescaped-entities */}
                  landscape, as well as recommendations from the WHO and global CDC's...
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  );
};

export default DashboardNewsWidgets;
