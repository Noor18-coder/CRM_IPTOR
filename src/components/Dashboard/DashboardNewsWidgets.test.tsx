import renderer from 'react-test-renderer';
import DashboardNewsWidgets from './DashboardNewsWidgets';

describe('[Dashboard] DashboardNewsWidgets', () => {
  it('should renders correctly', () => {
    const tree = renderer.create(<DashboardNewsWidgets />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
