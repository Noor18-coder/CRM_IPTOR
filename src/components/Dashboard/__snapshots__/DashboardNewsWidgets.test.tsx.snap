// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Dashboard] DashboardNewsWidgets should renders correctly 1`] = `
<div
  className="row newsandupdates"
>
  <div
    className="col-md-6 lft-col"
  >
    <div
      className="col-12 section-title"
    >
      <p>
        News & Updates
      </p>
    </div>
    <div
      className="news-container"
    >
      <ul
        className="news-collections-list"
      >
        <li>
          <div
            className="month-val"
          >
            August 2021
          </div>
          <ul
            className="news-feeds-items"
          >
            <li
              className="news-feed"
            >
              <p
                className="title"
              >
                Product launch
              </p>
              <p
                className="desc"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati animi labore ea, mollitia nemo, recusandae voluptatibus vero dolores assumenda nihil dolorem esse adipisci nobis natus tempore consequatur quas. Quia, doloribus.
              </p>
              <div
                className="d-flex justify-content-end"
              >
                <button
                  className="btn btn-link viewmore"
                  type="button"
                >
                  View more
                </button>
              </div>
            </li>
            <li
              className="news-feed"
            >
              <p
                className="title"
              >
                Product launch
              </p>
              <p
                className="desc"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati animi labore ea, mollitia nemo, recusandae voluptatibus vero dolores assumenda nihil dolorem esse adipisci nobis natus tempore consequatur quas. Quia, doloribus.
              </p>
              <div
                className="d-flex justify-content-end"
              >
                <button
                  className="btn btn-link viewmore"
                  type="button"
                >
                  View more
                </button>
              </div>
            </li>
            <li
              className="news-feed"
            >
              <p
                className="title"
              >
                Product launch
              </p>
              <p
                className="desc"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati animi labore ea, mollitia nemo, recusandae voluptatibus vero dolores assumenda nihil dolorem esse adipisci nobis natus tempore consequatur quas. Quia, doloribus.
              </p>
              <div
                className="d-flex justify-content-end"
              >
                <button
                  className="btn btn-link viewmore"
                  type="button"
                >
                  View more
                </button>
              </div>
            </li>
            <li
              className="news-feed"
            >
              <p
                className="title"
              >
                Product launch
              </p>
              <p
                className="desc"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati animi labore ea, mollitia nemo, recusandae voluptatibus vero dolores assumenda nihil dolorem esse adipisci nobis natus tempore consequatur quas. Quia, doloribus.
              </p>
              <div
                className="d-flex justify-content-end"
              >
                <button
                  className="btn btn-link viewmore"
                  type="button"
                >
                  View more
                </button>
              </div>
            </li>
          </ul>
        </li>
        <li>
          <div
            className="month-val"
          >
            September 2021
          </div>
          <ul
            className="news-feeds-items"
          >
            <li
              className="news-feed"
            >
              <p
                className="title"
              >
                Product launch
              </p>
              <p
                className="desc"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati animi labore ea, mollitia nemo, recusandae voluptatibus vero dolores assumenda nihil dolorem esse adipisci nobis natus tempore consequatur quas. Quia, doloribus.
              </p>
              <div
                className="d-flex justify-content-end"
              >
                <button
                  className="btn btn-link viewmore"
                  type="button"
                >
                  View more
                </button>
              </div>
            </li>
            <li
              className="news-feed"
            >
              <p
                className="title"
              >
                Product launch
              </p>
              <p
                className="desc"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati animi labore ea, mollitia nemo, recusandae voluptatibus vero dolores assumenda nihil dolorem esse adipisci nobis natus tempore consequatur quas. Quia, doloribus.
              </p>
              <div
                className="d-flex justify-content-end"
              >
                <button
                  className="btn btn-link viewmore"
                  type="button"
                >
                  View more
                </button>
              </div>
            </li>
            <li
              className="news-feed"
            >
              <p
                className="title"
              >
                Product launch
              </p>
              <p
                className="desc"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati animi labore ea, mollitia nemo, recusandae voluptatibus vero dolores assumenda nihil dolorem esse adipisci nobis natus tempore consequatur quas. Quia, doloribus.
              </p>
              <div
                className="d-flex justify-content-end"
              >
                <button
                  className="btn btn-link viewmore"
                  type="button"
                >
                  View more
                </button>
              </div>
            </li>
            <li
              className="news-feed"
            >
              <p
                className="title"
              >
                Product launch
              </p>
              <p
                className="desc"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati animi labore ea, mollitia nemo, recusandae voluptatibus vero dolores assumenda nihil dolorem esse adipisci nobis natus tempore consequatur quas. Quia, doloribus.
              </p>
              <div
                className="d-flex justify-content-end"
              >
                <button
                  className="btn btn-link viewmore"
                  type="button"
                >
                  View more
                </button>
              </div>
            </li>
            <li
              className="news-feed"
            >
              <p
                className="title"
              >
                Product launch
              </p>
              <p
                className="desc"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati animi labore ea, mollitia nemo, recusandae voluptatibus vero dolores assumenda nihil dolorem esse adipisci nobis natus tempore consequatur quas. Quia, doloribus.
              </p>
              <div
                className="d-flex justify-content-end"
              >
                <button
                  className="btn btn-link viewmore"
                  type="button"
                >
                  View more
                </button>
              </div>
            </li>
            <li
              className="news-feed"
            >
              <p
                className="title"
              >
                Product launch
              </p>
              <p
                className="desc"
              >
                Lorem ipsum dolor sit amet consectetur adipisicing elit. Obcaecati animi labore ea, mollitia nemo, recusandae voluptatibus vero dolores assumenda nihil dolorem esse adipisci nobis natus tempore consequatur quas. Quia, doloribus.
              </p>
              <div
                className="d-flex justify-content-end"
              >
                <button
                  className="btn btn-link viewmore"
                  type="button"
                >
                  View more
                </button>
              </div>
            </li>
          </ul>
        </li>
      </ul>
    </div>
  </div>
  <div
    className="col-md-6 rgt-col"
  >
    <div
      className="col-12 section-title"
    >
      <p>
        Calendar Events
      </p>
    </div>
    <div
      className="calendar-container"
    >
      <ul
        className="calendar-events-list"
      >
        <li>
          <div
            className="date-val"
          >
            6 August 2021
          </div>
          <ul
            className="cal-events-items"
          >
            <li
              className="events-list"
            >
              <p
                className="date-time"
              >
                04:00 pm - 05:00 pm
              </p>
              <p
                className="title"
              >
                Architecto pariatur eaque sed deserunt, ducimus doloremque.
              </p>
              <p
                className="desc"
              >
                Online Webex Meeting
              </p>
            </li>
            <li
              className="events-list"
            >
              <p
                className="date-time"
              >
                05:00 pm - 05:30 pm
              </p>
              <p
                className="title"
              >
                Expedita veritatis debitis eveniet a ratione facilis doloribus asperiores sit nemo.
              </p>
              <p
                className="desc"
              >
                Online Webex Meeting
              </p>
            </li>
          </ul>
        </li>
        <li>
          <div
            className="date-val"
          >
            5 August 2021
          </div>
          <ul
            className="cal-events-items"
          >
            <li
              className="events-list"
            >
              <p
                className="date-time"
              >
                09:00 am - 11:00 am
              </p>
              <p
                className="title"
              >
                CRM Project Essential Planning
              </p>
              <p
                className="desc"
              >
                Online Webex Meeting
              </p>
            </li>
            <li
              className="events-list"
            >
              <p
                className="date-time"
              >
                11:00 am - 11:30 am
              </p>
              <p
                className="title"
              >
                Lorem ipsum dolor sit amet.
              </p>
              <p
                className="desc"
              >
                Online Webex Meeting
              </p>
            </li>
            <li
              className="events-list"
            >
              <p
                className="date-time"
              >
                04:00 pm - 05:00 pm
              </p>
              <p
                className="title"
              >
                Architecto pariatur eaque sed deserunt, ducimus doloremque.
              </p>
              <p
                className="desc"
              >
                Online Webex Meeting
              </p>
            </li>
            <li
              className="events-list"
            >
              <p
                className="date-time"
              >
                05:00 pm - 05:30 pm
              </p>
              <p
                className="title"
              >
                Expedita veritatis debitis eveniet a ratione facilis doloribus asperiores sit nemo.
              </p>
              <p
                className="desc"
              >
                Online Webex Meeting
              </p>
            </li>
          </ul>
        </li>
        <li>
          <div
            className="date-val"
          >
            28 July 2021
          </div>
          <ul
            className="cal-events-items"
          >
            <li
              className="events-list"
            >
              <p
                className="date-time"
              >
                05:00 pm - 05:30 pm
              </p>
              <p
                className="title"
              >
                Expedita veritatis debitis eveniet a ratione facilis doloribus asperiores sit nemo.
              </p>
              <p
                className="desc"
              >
                Online Webex Meeting
              </p>
            </li>
            <li
              className="events-list"
            >
              <p
                className="date-time"
              >
                04:00 pm - 05:00 pm
              </p>
              <p
                className="title"
              >
                Architecto pariatur eaque sed deserunt, ducimus doloremque.
              </p>
              <p
                className="desc"
              >
                Online Webex Meeting
              </p>
            </li>
          </ul>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
